main.py

```diff
@@ -0,0 +1,79 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+import time
+
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+class QAAutomation:
```

Comment:
Inharit with `unittest.TestCase`. You can use it's functionalities for this task easily.